amundsen_application/static/js/components/HomePage/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
```

Comment:
nit: follow a loose import hierarchy of 3rd party imports (e.g. react/redux) --> absolute imports (e.g. components/ducks) --> relative imports (e.g. ../someFile)